Path-Check/covid-safe-paths
# Feature/add beta badge

#### Description:
Add a beta badge for iOS and Android for the App Store and Google Play Store.
Add tooling to fastlane to generate the badges.

#### Linked issues:

https://pathcheck.atlassian.net/browse/SAF-248

#### Screenshots:


#### How to test:
Install a test APK or IPA on a device. Validate that the icon rendered on the device has a diagonal stripe over the icon in dark grey, with the word `beta` written on it (see screenshots)
